dev-lang/elm-compiler-bin/elm-compiler-bin-0.19.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The 'elm' command line interface (binary)"
+HOMEPAGE="http://elm-lang.org"
+SRC_URI="https://github.com/elm/compiler/releases/download/${PV}/binary-for-linux-64-bit.gz -> ${PV}.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="+symlink"
+
+RDEPEND=" >=dev-haskell/ansi-terminal-0.8:=
+	>=dev-haskell/ansi-wl-pprint-0.6.8:= <dev-haskell/ansi-wl-pprint-0.7:=
+	>=dev-haskell/edit-distance-0.2:= <dev-haskell/edit-distance-0.3:=
+	dev-haskell/file-embed:=
+	dev-haskell/filelock:=
+	dev-haskell/haskeline:=
+	>=dev-haskell/http-4000.2.5:= <dev-haskell/http-4000.4:=
+	>=dev-haskell/http-client-0.6:= <dev-haskell/http-client-0.7:=
+	>=dev-haskell/http-client-tls-0.3:= <dev-haskell/http-client-tls-0.4:=
+	>=dev-haskell/http-types-0.12:= <dev-haskell/http-types-1.0:=
+	>=dev-haskell/language-glsl-0.3:=
+	>=dev-haskell/mtl-2.2.1:= <dev-haskell/mtl-3:=
+	>=dev-haskell/network-2.4.0.0:=
+	dev-haskell/parsec:=
+	dev-haskell/raw-strings-qq:=
+	dev-haskell/scientific:=
+	dev-haskell/sha:=
+	dev-haskell/snap-core:=
+	dev-haskell/snap-server:=
+	dev-haskell/unordered-containers:=
+	dev-haskell/utf8-string:=
+	dev-haskell/vector:=
+	dev-haskell/zip-archive:=
+	>=dev-lang/ghc-8.2.1:=
+"
+
+RDEPEND=" symlink? ( !dev-lang/elm-compiler )"
```

Comment:
`elm-compiler` would also need the same `!` blocker against this package.

Reply:
I'll leave this one as `[draft]` until the other PR goes through, so that I can then make this change you've requested.